components/pagination/stories/template.js

```diff
@@ -0,0 +1,88 @@
+import { html } from "lit-html";
+import { classMap } from "lit-html/directives/class-map.js";
+import { repeat } from 'lit-html/directives/repeat.js';
+
+import { Template as Button } from "@spectrum-css/button/stories/template.js";
+import { Template as ActionButton } from "@spectrum-css/actionbutton/stories/template.js";
+import { Template as Textfield } from "@spectrum-css/textfield/stories/template.js";
+
+
+import '../index.css';
+import '../skin.css';
+
+// More on component templates: https://storybook.js.org/docs/web-components/writing-stories/introduction#using-args
+export const Template = ({
+  rootClass = "spectrum-Pagination",
+  size = "m",
+  customClasses = [],
+  variant,
+  items,
+  ...globals
+}) => {
+
+  if (variant === 'explicit') {
+    return html`
+      <nav class=${classMap({
+        [rootClass]: true,
+        [`${rootClass}--${variant}`]: typeof variant !== "undefined",
```

Comment:
```suggestion
        [`${rootClass}--explicit`]: true,
```

Since this is wrapped in an if statement checking that it equals explicit, probably safe to hardcode this here for clarity.